test-harness/src/test/java/io/jenkins/plugins/casc/core/ProxyConfiguratorTest.java

```diff
@@ -75,9 +75,9 @@ public void shouldSetProxyWithMinimumFields() throws Exception {
         final CNode node = c.describe(proxy, context);
         assertNotNull(node);
         Mapping mapping = node.asMapping();
-        assertEquals(3, node.asMapping().size());
+        assertEquals(2, node.asMapping().size());
         assertEquals("proxyhost", mapping.getScalarValue("name"));
-        assertEquals("", Secret.decrypt(mapping.getScalarValue("password")).getPlainText());
+        assertEquals("80", mapping.getScalarValue("port"));
```

Comment:
What changed? 🤔

Reply:
https://github.com/jenkinsci/jenkins/pull/3935

Reply:
great! I was looking for that PR, but I didn't find it